### PR TITLE
README: Add entries for TiDB Dev Guide and TiDB Internals Forum

### PR DESCRIPTION
**Pull request by tisonkun**
### Release note

```release-note
None
```

### CODE REVIEW

**Comment by sunxiaoguang** [comment]
LGTM

**Review by fgksgf — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -46,9 +46,9 @@ For more details and latest updates, see [TiDB docs](https://docs.pingcap.com/ti
 
 You can join these groups and chats to discuss and ask TiDB related questions:
 
-- [Contributors Mailing list](https://lists.tidb.io/g/contributors)
+- [TiDB Internals Forum](https://internals.tidb.io/)
 - [Slack Channel](https://slack.tidb.io/invite?team=tidb-community&channel=everyone&ref=pingcap-tidb)
-- [Chinese Forum](https://asktug.com)
+- [Chinese User Forum](https://asktug.com)
```
Comment (zz-jason) [inline README.md:51]:
```suggestion
- [TiDB User Group Forum (Chinese)](https://asktug.com)
```

**Review by zz-jason — COMMENTED** [review]
/LGTM

**Review by zz-jason — APPROVED** [review]

**Review by winkyao — APPROVED** [review]
LGTM

**Pull request merged**